src/pages/Encounters/EncounterHistorySelector.tsx

```diff
@@ -412,23 +465,68 @@ const EncounterSheetTrigger = () => {
                 </span>
               )}
             </span>
-            <div className="w-px h-4 bg-gray-300 hidden sm:block" />
-            <span className="text-sm text-gray-700">
-              {encounter.facility.name}
-            </span>
+            <div className="flex gap-3 items-center">
+              <Badge
+                variant={ENCOUNTER_STATUS_COLORS[encounter.status]}
+                size="sm"
+                className=" whitespace-nowrap"
+              >
+                {t(`encounter_status__${encounter.status}`)}
+              </Badge>
+              <div
+                className={buttonVariants({ variant: "ghost", size: "icon" })}
+              >
+                <ChevronDown />
+              </div>
+            </div>
           </div>
         </div>
-        <div className="flex gap-3 items-center">
-          <div className="my-2">
-            <Badge variant={ENCOUNTER_STATUS_COLORS[encounter.status]}>
-              {t(`encounter_status__${encounter.status}`)}
-            </Badge>
-          </div>
-          <div className={buttonVariants({ variant: "ghost", size: "icon" })}>
-            <ChevronDown />
+        {encounter.tags.length > 0 && (
+          <div className="flex flex-wrap gap-2">
+            {encounter.tags.map((tag) => (
+              <Badge
+                key={tag.id}
+                variant="secondary"
+                className="capitalize"
+                title={tag.description}
+              >
+                {getTagHierarchyDisplay(tag)}
+              </Badge>
+            ))}
           </div>
-        </div>
+        )}
       </CardContent>
     </Card>
   );
 };
+
+const EncounterHoverCard = ({ encounter }: { encounter: EncounterRead }) => {
```

Comment:
component naming is wrong now right? 🤔 

1. the hover card is now hovering on top of the tags text only right? and not the entire encounter card right? 

2. now that it's very minimal / small content, i'd prefer keeping the contents in-lined itself.

```suggestion
const EncounterTagHoverCard = ({ encounter }: { encounter: EncounterRead }) => {
```